person.rb

```diff
@@ -1,7 +1,8 @@
+require_relative 'rental'
```

Comment:
**[Optional]**

- [ ] It would be nice if you rename **rental** to **rentals** because you have a file with an `s` at the end so it is breaking the operation :thinking: and you are calling it with the wrong name means without `s` at the end :fearful: Kindly put an `s` at the end of `require_relative 'rental'`